sonar-kotlin-api/build.gradle.kts

```diff
@@ -37,3 +37,10 @@ tasks.withType<Javadoc> {
         addStringOption("tag", "javax.annotation.ParametersAreNonnullByDefault:ParametersAreNonnullByDefault")
     }
 }
+
+task<JavaExec>("printAst") {
+    dependsOn(":sonar-kotlin-plugin:compileJava")
```

Comment:
As discussed in person, after moving it outside of the `utils-kotlin` module, I don't think we need this dependency; we're not even sure we needed it in the first place

Reply:
Done